### PR TITLE
Add CSS FILE

### PR DESCRIPTION
**Pull request by Raphael2k6**

### CODE REVIEW

**Comment by Temitoyin** [comment]
This is good work Raphael, The mobile view looks nice, but the desktop view has some alignment issues, which could be improved on. I also think a different font selection would have a better effect on the page.

File: style.css [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+    justify-content: center;
+    font-size: 1rem;
+    margin: auto;
+    background-color: #17252A;
+}
+
+.main-nav li {
+    padding: 2rem;
+    list-style: none;
+    margin-right: 30px;
```
Comment (CodyGbengz) [inline style.css]: Use `rem` please.

File: style.css [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+}
+
+.main-nav li:hover {
+   color: #2B7A78;
```
Comment (CodyGbengz) [inline style.css]: For colors used more than once, consider using variables.

File: style.css [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+    color: #FEFFFF;
+    font-size: 5rem;
+}
+
+.about-me, .contact {
+    color: #17252A;
+    background-color: #DEF2F1;
+    padding: 5px 10px 5px 10px;
+    text-align: center;
+    font-family: Sentinel SSm A,Sentinel SSm B,system-ui,-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Helvetica,Arial,sans-serif,Apple Color Emoji,Segoe UI Emoji,Segoe UI Symbol;
```
Comment (CodyGbengz) [inline style.css]: You can create a subclass and include this property. Then add this class to any element or component that needs this. Less repetition of code.

File: style.css [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+    background-color: #17252A;
+    padding: 10px 10px 20px 20px;
+    color: #FEFFFF;
+    font-family: Ringside Regular A, Ringside Regular B,Rubik,Lato,Lucida Grande,Lucida Sans Unicode,Tahoma,Sans-Serif;
+}
+
+.email {
+    padding: 5px 50px;
+    margin-left: 5px;
+    border-radius: 10px;
```
Comment (CodyGbengz) [inline style.css]: Use `rem`

File: style.css [hunk 5]
```diff
@@ -0,0 +1,131 @@
...
+    background-color: #DEF2F1;
+}
+
+.btn--color {
+    color: #FEFFFF;
+    background-color: #2B7A78;
+    border-radius: 25px;
+    margin-left: 5px;
+    border: 1px solid #2B7A78;
+    padding: 5px 20px 5px 20px;
```
Comment (CodyGbengz) [inline style.css]: `padding: 5px 20px` will suffice. You can consider using `rem` as well.

File: style.css (hunk 1 above)
Comment (Raphael2k6) [reply style.css]: I have modified this, thanks.

File: style.css (hunk 2 above)
Comment (Raphael2k6) [reply style.css]: Modified

File: style.css (hunk 5 above)
Comment (Raphael2k6) [reply style.css]: Done

File: style.css (hunk 4 above)
Comment (Raphael2k6) [reply style.css]: Done

File: style.css (hunk 3 above)
Comment (Raphael2k6) [reply style.css]: done

File: _navbar.scss [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+$mainWhite: #FEFFFF;
+$primaryFont: Sentinel SSm A,Sentinel SSm B,system-ui,-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Helvetica,Arial,sans-serif,Apple Color Emoji,Segoe UI Emoji,Segoe UI Symbol;
+
+nav {
+    ul {
+        font-family: $primaryFont;
+        display: flex;
+        flex-flow: row wrap;
+        justify-content: center;
+        font-size: 1rem bold;
```
Comment (CodyGbengz) [inline _navbar.scss]: This doesn't look right. `bold` doesn't seem like a valid value for font-size.

File: styles.scss [hunk 7]
```diff
@@ -0,0 +1,127 @@
+@use 'navbar';
+@use 'main-display';
+
+$mainColor: rgb(26, 27, 29);
+$effectColor: rgb(202, 128, 17);
+$mainWhite: #FEFFFF;
```
Comment (CodyGbengz) [inline styles.scss]: Please be consistent with your approach. RGB or Hex code for colors? choose one and stick to it. There are lots of color tools that can help with conversion.

File: _navbar.scss (hunk 6 above)
Comment (Raphael2k6) [reply _navbar.scss]: Ok, fixed.

**Pull request merged**